UI/Web/src/app/reading-list/_components/reading-list-item/reading-list-item.component.html

```diff
@@ -42,7 +42,7 @@ <h5 class="mb-1 pb-0" id="item.id--{{position}}">
 
         @if (item.releaseDate !== '0001-01-01T00:00:00') {
           <div class="ps-1 mt-2">
-            Released: {{item.releaseDate | date:'longDate'}}
+            {{t('released')}}{{item.releaseDate | date:'longDate'}}
```

Comment:
You need to update this to be `{{t('released-label', {date: item.releaseDate | date:'longDate'})}}`. 

This is because not all languages are read left to right or even have spaces. Kavita has some norms for localizers on how strings are used based on their key name.